.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "restructuredtext.confPath": "c:\\Users\\faoustin\\Downloads\\pgcli"
+}
```

Comment:
Was committing this file an accident? Maybe add `.vscode` to your personal gitignore file?